### ISSUE TITLE
LPIPS evaluation problem

### ISSUE THREAD

**Issue by JianqiangRen** [xg-chu/GAGAvatar#51]
@xg-chu Thank you for your great work on GAGAvatar!

I am currently evaluating the model performance using the pretrained weights provided at https://huggingface.co/xg-chu/GAGAvatar/resolve/main/assets/GAGAvatar.pt, and calculating the LPIPS metric on the VFHQ test set, which consists of 50 clips, 50 samples per clip (a total of 2500 source-GT image pairs).

The test images were preprocessed by cropping and resizing to 512 × 512 resolution, followed by track_lmdb processing.

However, I am unable to reproduce the reported LPIPS score of 0.122 from your paper.

Could you please clarify the following two points?

1. Which version of the LPIPS library did you use? Was it v0.0 or v0.1?

2. Were the input images normalized to the range [-1, 1] before computing LPIPS?

For reference, here is the evaluation code I used:
```
import lpips
loss_fn = lpips.LPIPS(net='alex')

render_results = model.forward_expression(one_data)
gt_rgb = render_results['t_image'].clamp(0, 1)
pred_rgb = render_results['gen_image'].clamp(0, 1)
pred_sr_rgb = render_results['sr_gen_image'].clamp(0, 1) 

pred_normalized = pred_sr_rgb *2 -1
gt_normalized = gt_rgb *2 -1
lpips_value = loss_fn.forward(pred_normalized, gt_normalized) #   rgb images scaled in [-1, +1]
```
Any additional guidance on preprocessing or implementation details would be greatly appreciated.

Best regards,

**Comment by xg-chu** [comment]
What LPIPS did you get? The released version should perform better than the paper version.

**Comment by xg-chu** [comment]
Here is the LPIPS code, the LPIPS of the released model should be 0.0922.
```
@torch.no_grad()
def calc_lpips(preds, target):
    # input should be in range [0, 1]
    lpips_model = torchmetrics.image.lpip.LearnedPerceptualImagePatchSimilarity(net_type='squeeze')
    img1 = (preds * 2) - 1; img2 = (target * 2) - 1
    lpips = lpips_model(img1[None], img2[None])
    return lpips
```

**Comment by JianqiangRen** [comment]
> Here is the LPIPS code, the LPIPS of the released model should be 0.0922.
> 
> ```
> @torch.no_grad()
> def calc_lpips(preds, target):
>     # input should be in range [0, 1]
>     lpips_model = torchmetrics.image.lpip.LearnedPerceptualImagePatchSimilarity(net_type='squeeze')
>     img1 = (preds * 2) - 1; img2 = (target * 2) - 1
>     lpips = lpips_model(img1[None], img2[None])
>     return lpips
> ```

I have found the reason: you used the squeeze network for LPIPS, while I was using alex.
I just re-ran the test, and the results are as follows:

LPIPS (squeeze): 0.0944
LPIPS (alex):  0.1394

**Comment by xg-chu** [comment]
All the numbers in paper use the same squeeze network.
